mne/viz/_3d.py

```diff
@@ -1517,13 +1517,17 @@ def _plot_mpl_stc(stc, subject=None, surface='inflated', hemi='lh',
     return fig
 
 
-def link_brains(brains):
+def link_brains(brains, time=True, camera=False):
     """Plot multiple SourceEstimate objects with PyVista.
 
     Parameters
     ----------
     brains : list, tuple or np.ndarray
         The collection of brains to plot.
+    time : bool
+        If True, link the time controllers. Defaults to True.
+    camera : bool
+        If True, link the camera controls. Defaults to True.
```

Comment:
```suggestion
        If True, link the camera controls. Defaults to False.
```
Or change it above, in which case `_LinkViewer.__init__` should also change